src/Bicep.LangServer/Telemetry/BicepTelemetryEvent.cs

```diff
@@ -111,14 +111,15 @@ public static BicepTelemetryEvent CreateDisableNextLineDiagnostics(string code)
                 }
             );
 
-        public static BicepTelemetryEvent EditLinterRule(string code, bool newConfigFile, string? error)
+        public static BicepTelemetryEvent EditLinterRule(string code, bool newConfigFile, bool newRuleAdded, string? error)
             => new BicepTelemetryEvent
             (
                 eventName:  TelemetryConstants.EventNames.EditLinterRule,
                 properties: new()
                 {
                     ["code"] = code,
-                    ["newConfigFile"] = ToString(newConfigFile),
+                    ["newConfigFile"] = ToTrueFalse(newConfigFile),
```

Comment:
Is there a reason why we don't want to use Boolean.ToString method here?
["newConfigFile"] = newConfigFile.ToString(),
["newRuleAdded"] = newRuleAdded.ToString(),

Reply:
Casing.  I'd rather keep it consistently lower-case which is what everything else uses that comes from the extension.

Reply:
ToString() gives upper-case.

Reply:
Sounds good.